More WS SVGs

sareh opened the pull request:
Resolves #635

**Overall change:** Add Mundo and Punjabi SVGs. Rename Turkish->Turkce SVG. Add export & test for Portuguese SVG. 
Update all SVGs with a fix for the outline - white border, not a black border. 

**Mundo**
http://localhost:8180/?path=/story/utilities-svgs--mundo
![Screen Shot 2019-08-06 at 17 31 42](https://user-images.githubusercontent.com/3028997/62558091-3b2a8100-b870-11e9-86c1-daa3e5419a94.png)

**Portuguese**
http://localhost:8180/?path=/story/utilities-svgs--portuguese
![Screen Shot 2019-08-06 at 17 31 47](https://user-images.githubusercontent.com/3028997/62558089-3b2a8100-b870-11e9-9e87-bc3925ffd892.png)

**Punjabi**
http://localhost:8180/?path=/story/utilities-svgs--punjabi
![Screen Shot 2019-08-06 at 17 31 53](https://user-images.githubusercontent.com/3028997/62558088-3a91ea80-b870-11e9-9e68-5f74d7c1cd69.png)

**Turkce**
http://localhost:8180/?path=/story/utilities-svgs--turkce
![Screen Shot 2019-08-06 at 17 32 03](https://user-images.githubusercontent.com/3028997/62558087-3a91ea80-b870-11e9-87bc-3ddd4fda0691.png)
**Code changes:**

- Add Mundo and Punjabi SVGs. 
- Rename Turkish->Turkce SVG
- Add export & test for Portuguese svg
- Update border style.

Changing the background colour of the story:
<img width="361" alt="Screen Shot Mundo SVG background colour postbox" src="https://user-images.githubusercontent.com/3028997/62616516-ab381600-b907-11e9-8e42-98fad890a876.png">

Without CSS, the Mundo SVG:
<img width="274" alt="Screen Shot Mundo SVG without CSS" src="https://user-images.githubusercontent.com/3028997/62616462-7a57e100-b907-11e9-9dff-d7797af1b2af.png">


---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

sareh committed:
Add Mundo and Punjabi SVGs. Rename Turkish->Turkce SVG

sareh committed:
Changelog and version update

sareh committed:
Add export & test for portuguese svg

Commit:
Merge branch 'latest' into more-ws-svgs

sareh commented on packages/utilities/psammead-assets/CHANGELOG.md:
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.0.0   | [PR#1646](https://github.com/bbc/psammead/pull/1646) Add SVGs for these services mundo, punjabi. Update SVG name from turkish to turkce.  |
```
This is a major version bump since an export had to be renamed from `turkish` to `turkce`, which is the service name. `https://www.bbc.com/turkce`

chris-hinds reviewed (commented):
Should these new SVG's be visible in the brand container when viewing in storybook?

sareh commented:
@hindsc52 Not yet - when this `psammead-assets` package is published, then I will update it as a dependency within the `psammead-brand` so these new SVGs are visible in the Brand Storybook story.

chris-hinds commented:
> @hindsc52 Not yet - when this `psammead-assets` package is published, then I will update it as a dependency within the `psammead-brand` so these new SVGs are visible in the Brand Storybook story.

Got it, Thank you.

chris-hinds reviewed (approved):
👍

sareh committed:
Update dependency version

dr3 reviewed (approved):
LGTM, make sure to a11y check them before merge if we havent already :)

sareh committed:
Move style stroke #fff from the pipe symbol to around the whole svg group

sareh committed:
Return background to black for stories

sareh committed:
Changelog update

sareh commented:
@dr3 & @hindsc52 I've updated the outline styles of the SVGs, since they were black instead of white and added extra screenshots to the PR description.

Commit:
Merge branch 'latest' into more-ws-svgs

sareh committed:
Add Japanese SVG

Commit:
Merge branch 'latest' into more-ws-svgs

sareh commented:
All SVGs have been UX reviewed by Tom. Checked with the Postbox red background, with CSS disabled, changing colours in Firefox Preferences, and in IE high contrast mode.

dr3 reviewed (commented):
Hey :) howcome youve used 
```jsx
<g fillRule="evenodd" stroke="#000" strokeWidth=".335" style={{ stroke: '#fff' }} >
```
not
```jsx
<g fillRule="evenodd" stroke="#fff" strokeWidth=".335" >
```

sareh committed:
Lint fix

Commit:
Merge branch 'latest' into more-ws-svgs

sareh commented:
@dr3 It was as a result of the accessibility testing. It is to ensure that when CSS is disabled, you don't have a white logo on a white background, but you have a black outline to the logo.

Commit:
Merge branch 'latest' into more-ws-svgs

sareh commented:
Approved the Chromatic QA snapshots in Chrome and IE11. 
Merging now.
